notebooks/306-nncf-with-nlp-huggingface-transformers/bert_mrpc_int8.yml

```diff
@@ -0,0 +1,39 @@
+models:
+  - name: bert-mrpc
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+            type: classification
+            num_classes: 2
+        inputs:
+          - name: "result.1"
+            type: INPUT
+            value: "input_ids"
+          - name: "result.3"
+            type: INPUT
+            value: 'input_mask'
+          - name: 'result.5'
+            type: INPUT
+            value: 'segment_ids'
+        model: bert_mrpc_int8.xml
+        weights: bert_mrpc_int8.bin
+
+    datasets:
+      - name: mrpc
+        data_source: ./MRPC
+        annotation_conversion:
+            converter: mrpc
+            annotation_file: ./MRPC/dev.tsv
+            vocab_file: ../vocab.txt
```

Comment:
if vocab.txt requires only for accuracy measurement, you can use original HF tokenizer with providing model_id without attaching vocab file

Reply:
@vshampor, Could you please take a look at @eaidova's comments? Appreciate it.